<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCourierItem</class>
 <widget class="QWidget" name="NewCourierItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>790</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>790</width>
    <height>82</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#NewCourierItem {
    background: transparent;
    border-radius: 8px;
}

QLabel#labMusicNum {
    color: rgb(195, 195, 219);
}

QLabel#labMusicLog
{
	border-image: url(:/NewMusic/images/NewMusic/sq2.png);
}

QLabel#MusicName {
    color: rgb(54, 54, 54);
}

QLabel#MusicAuthor,
#MusicAlbum {
    color: rgb(101, 101, 101);
}

QLabel#MusicTimer {
    color: rgb(155, 155, 202);
}</string>
  </property>
  <widget class="QLabel" name="labMusicNum">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <property name="text">
    <string>01</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widMusicPic" native="true">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>11</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>363</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="MusicAuthor">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
       </font>
      </property>
      <property name="text">
       <string>MusicAuthor</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>78</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="MusicAlbum">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
       </font>
      </property>
      <property name="text">
       <string>MusicAlbum</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>48</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="MusicTimer">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
       </font>
      </property>
      <property name="text">
       <string>MusicTimer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="MusicName">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
       </font>
      </property>
      <property name="text">
       <string>MusicName</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="labMusicLog">
      <property name="minimumSize">
       <size>
        <width>22</width>
        <height>17</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>27</width>
        <height>17</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
